.github/workflows/onPushToMain.yml

```diff
@@ -9,36 +9,32 @@ jobs:
     uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
   release:
     runs-on: ubuntu-latest
-    container:
-      image: node:lts
     needs:
     - tests
     steps:
-    - uses: actions/checkout@v3
-      with:
-        token: ${{ secrets.IDEE_GH_TOKEN }}
-    - uses: ./.github/actions/gitConfig
-      with:
-        email: ${{ secrets.IDEE_GH_EMAIL }}
-    - run: yarn
-    - name: Conventional Changelog Action
-      id: changelog
-      uses: TriPSs/conventional-changelog-action@ed98b658f3015c5afb96a13caf9bb81f69df49e6
-      with:
-        git-user-name: Release Bot
-        git-user-email: ${{ secrets.IDEE_GH_EMAIL }}
-        github-token: ${{ secrets.IDEE_GH_TOKEN }}
-        skip-version-file: true
-        output-file: false
-        tag-prefix: "v"
```

Comment:
From this section, I removed tag-prefix and fallback version (which was necessary for the first run, but isn't any longer).